ale/drivers/__init__.py

```diff
@@ -130,3 +144,30 @@ def loads(label, props='', formatter='ale', verbose=False):
     """
     res = load(label, props, formatter, verbose=verbose)
     return json.dumps(res, cls=AleJsonEncoder)
+
+
+def parse_label(label, grammar=pvl.grammar.PVLGrammar):
+    """
+    Attempt to parse a PVL label.
+
+    Returns
```

Comment:
This should be Parameters, and this docs section also needs to define a return type. Load and Loads are good examples to draw from. If the label is able to be parsed the function should return a string